sql/core/src/test/resources/sql-tests/inputs/postgreSQL/window_part1.sql

```diff
@@ -301,7 +301,7 @@ FROM tenk1 WHERE unique1 < 10;
 -- unique1, four
 -- FROM tenk1 WHERE unique1 < 10 WINDOW w AS (order by four);
 
--- [SPARK-27951] ANSI SQL: NTH_VALUE function
+-- [SPARK-30708] first_value/last_value window function throws ParseException
```

Comment:
@beliefer, why was SPARK-30708 resolved? Seems like still a valid issue.

Reply:
Because #25082 has reverted, SPARK-30708 not need.